Modelica/Mechanics/Translational/Components/SupportFriction.mo

```diff
@@ -1,51 +1,71 @@
 within Modelica.Mechanics.Translational.Components;
 model SupportFriction "Coulomb friction in support"
-
   extends Modelica.Mechanics.Translational.Interfaces.PartialElementaryTwoFlangesAndSupport2;
   extends Modelica.Thermal.HeatTransfer.Interfaces.PartialElementaryConditionalHeatPortWithoutT;
 
+  import Modelica.Blocks.Types.{ExternalCombiTable1D, Extrapolation, Smoothness};
+  import Modelica.Blocks.Tables.Internal.{getTable1DValue, getTable1DValueNoDer, getTable1DValueNoDer2};
+
   parameter Real f_pos[:, 2]=[0, 1]
     "Positive sliding friction characteristic [N] as function of v [m/s] (v>=0)";
+  parameter Smoothness smoothness = Smoothness.LinearSegments
+    "Smoothness of interpolation in f_pos table";
   parameter Real peak(final min=1) = 1
     "Peak for maximum friction force at v==0 (f0_max = peak*f_pos[1,2])";
+
   extends Translational.Interfaces.PartialFriction;
 
   SI.Position s "= flange_a.s - support.s";
   SI.Force f "Friction force";
   SI.Velocity v "Absolute velocity of flange_a and flange_b";
   SI.Acceleration a "Absolute acceleration of flange_a and flange_b";
+
+protected
+  final parameter ExternalCombiTable1D tableID = ExternalCombiTable1D(
+    tableName="NoName",
+    fileName="NoName",
+    table=f_pos,
+    columns={2},
+    smoothness=smoothness,
+    extrapolation=Extrapolation.LastTwoPoints,
+    verboseRead=false) "External table object for sliding friction coefficient";
+
+  Real table_signs[2]
+    "Signs for sliding friction coefficient table interpolation: [sign for w_rel, sign for mu]";
```

Comment:
There is neither `w_rel` nor `mu` in SupportFriction.

Reply:
Fix with 9124e3f561ba0f418ba64028ed0c56e05157f778